srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java

```diff
@@ -167,7 +180,22 @@ public String getMeasureDate(final String key)
      */
     public void setMeasureDate(final String key, final String retrievalString)
     {
-        fDynamicValues.put(key + VariableEntry.SEPARATOR + VariableEntry.RETRIEVAL_STRING, retrievalString);
+        fDynamicValues.put(makeRetrievalInfoKey(key), retrievalString);
+    }
+    
+    /**
+     * Puts the given retrieved value into the dynamic values map under the given key,
+     * also populating the human-readable retrieval info.
+     * @param key the input key to populate
+     * @param retrievedValue the retrieved value
+     * @return {@code this} for convenience
```

Comment:
So what is more readable here, stating "this" or "the calling object"? Everyone understands "this" in the code, but is it weird to put it in the javadocs?